readline/line.go

```diff
@@ -183,26 +184,47 @@ func (rl *Instance) deleteToEnd() {
 	rl.line = rl.line[:rl.pos]
 }
 
-func (rl *Instance) emacsForwardWord(tokeniser tokeniser) (adjust int) {
-	// when emacs has more specific stuff, move this in a file with then
-	split, index, pos := tokeniser(rl.line, rl.pos)
-	if len(split) == 0 {
-		return
+func isAlNum(r rune) (b bool) {
```

Comment:
slight nitpick, function name is confusing

Reply:
used isalnum because c https://www.programiz.com/c-programming/library-function/ctype.h/isalnum#:~:text=The%20isalnum%20%28%29%20function%20checks%20whether%20the%20argument,argument%20is%20neither%20an%20alphabet%20nor%20a%20digit. can change if u want

Reply:
removed because now we use tokenizer